mullvad-types/src/relay_constraints.rs

```diff
@@ -523,8 +525,16 @@ impl fmt::Display for SelectedObfuscation {
 }
 
 #[derive(Default, Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
+#[cfg_attr(target_os = "android", derive(FromJava, IntoJava))]
+#[cfg_attr(target_os = "android", jnix(package = "net.mullvad.mullvadvpn.model"))]
 #[serde(rename_all = "snake_case")]
 pub struct Udp2TcpObfuscationSettings {
+    // TODO: don't ignore port set by JNI
+    #[cfg_attr(
+        target_os = "android",
+        jnix(map = "|constraint| constraint.map(|v| v as i32)")
+    )]
+    #[cfg_attr(target_os = "android", jnix(default))]
     pub port: Constraint<u16>,
```

Comment:
I think we should use `jnix(skip)` to ignore the field altogether until selecting a port is supported.